Package.swift

```diff
@@ -38,6 +38,7 @@ let package = Package(
             targets: [
                 "SourceControl",
                 "SPMLLBuild",
+		"PackageCollections",
```

Comment:
nit: indent

Reply:
Ah yes, this was somehow using tabs instead of spaces.